.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -11,25 +11,24 @@ body:
       - Search [open](https://github.com/godotengine/godot/issues) and [closed](https://github.com/godotengine/godot/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported. If you don't find a relevant match or if you're unsure, don't hesitate to **open a new issue**. The bugsquad will handle it from there if it's a duplicate.
       - Verify that you are using a [supported Godot version](https://docs.godotengine.org/en/stable/about/release_policy.html).
 
-- type: input
-  attributes:
-    label: Godot version
```

Comment:
I think we should keep the Godot version section separate, and it's okay if the auto-generated list includes it as well. Most users are going to report issues without using that editor button, and it's good to incentivize them to provide a concrete version number in an explicit way.

Reply:
> and it's okay if the auto-generated list includes it as well

do you mean the commit hash? Can I get that with 
https://github.com/godotengine/godot/blob/36e7800679e406ac79e6865cabaf0e7e69bc81d8/editor/editor_vcs_interface.cpp#L125-L136
?

Reply:
I mean that it's fine if we have a dedicated section for the Godot version AND if the "System Info" generated by the editor includes the version as well, as it does already with this PR.

Reply:
Roger

Reply:
Done